sdk/tests/conformance2/renderbuffers/multisample-draws-between-blits.html

```diff
@@ -0,0 +1,217 @@
+    uniform mat4 mat;
+
+    void main() {
+      gl_Position = mat * position;
+    }
+    `;
+
+    const fs = `#version 300 es
+    precision mediump float;
+    uniform vec4 color;
+    out vec4 outColor;
+    void main() {
+      outColor = color;
+    }
+    `;
+
+    const texVS = `#version 300 es
+
+    layout(location = 0) in vec4 position;
+    out vec2 texcoord;
+    uniform mat4 mat;
+
+    void main() {
+      gl_Position = mat * position;
+      texcoord = position.xy;
+    }
+    `;
+
+    const texFS = `#version 300 es
+    precision mediump float;
+  in vec2 texcoord;
+    uniform sampler2D tex;
+    out vec4 outColor;
+    void main() {
+      outColor = texture(tex, texcoord);
+    }
+    `;
+
+    const msRB = gl.createRenderbuffer();
+    gl.bindRenderbuffer(gl.RENDERBUFFER, msRB);
+    gl.renderbufferStorageMultisample(gl.RENDERBUFFER, 4, gl.RGBA8, w, h);
+
+    const msFB = gl.createFramebuffer();
+    gl.bindFramebuffer(gl.FRAMEBUFFER, msFB);
+    gl.framebufferRenderbuffer(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.RENDERBUFFER, msRB);
+
+    const dTex = gl.createTexture();
+    gl.bindTexture(gl.TEXTURE_2D, dTex);
+    gl.texStorage2D(gl.TEXTURE_2D, 1, gl.RGBA8, w, h);
+    gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.NEAREST);
+
+    const dFB = gl.createFramebuffer();
+    gl.bindFramebuffer(gl.DRAW_FRAMEBUFFER, dFB);
+    gl.framebufferTexture2D(gl.DRAW_FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, dTex, 0);
+
+    const positionLoc = 0;  // hard coded in shaders so they match
+    const buf = gl.createBuffer();
+    gl.bindBuffer(gl.ARRAY_BUFFER, buf);
+    gl.bufferData(gl.ARRAY_BUFFER, new Float32Array([
+      0, 0,
+      1, 0,
+      0, 1,
+      0, 1,
+      1, 0,
+      1, 1,
+    ]), gl.STATIC_DRAW);
+    gl.enableVertexAttribArray(positionLoc);
+    gl.vertexAttribPointer(positionLoc, 2, gl.FLOAT, false, 0, 0);
+
+    const program = wtu.setupProgram(gl, [vs, fs]);
+    const texProgram = wtu.setupProgram(gl, [texVS, texFS]);
+
+    const colorLoc = gl.getUniformLocation(program, 'color');
+    const matLoc = gl.getUniformLocation(program, 'mat');
+    const texMatLoc = gl.getUniformLocation(texProgram, 'mat');
+
+    gl.useProgram(program);
+
+    const drawAndResolve = (color, mat) => {
+      gl.bindFramebuffer(gl.FRAMEBUFFER, msFB);
+      gl.uniform4fv(colorLoc, color);
+      gl.uniformMatrix4fv(matLoc, false, mat);
+      gl.drawArrays(gl.TRIANGLES, 0, 6);
+
+      gl.bindFramebuffer(gl.DRAW_FRAMEBUFFER, dFB);
+      gl.blitFramebuffer(0, 0, w, h, 0, 0, w, h, gl.COLOR_BUFFER_BIT, gl.NEAREST);
+    };
+
+    const check = (x, y, w, h, expected, msg) => {
+      gl.bindFramebuffer(gl.FRAMEBUFFER, dFB);
+      wtu.checkCanvasRect(gl, x, y, w, h, expected, msg);
+    };
+
+    debug('fill with red');
+    drawAndResolve([1, 0, 0, 1], [
+      2, 0, 0, 0,
+      0, 2, 0, 0,
+      0, 0, 1, 0,
+      -1, -1, 0, 1,
+    ]);
+    check(0, 0, w, h, [255, 0, 0, 255], 'whole thing');
```

Comment:
Could you define variables for the colors? e.g.

`const red = [255, 0, 0, 255];`

Reply:
done